src/screens/TrackListScreen.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Text, StyleSheet, Button } from "react-native";
+
+const TrackListScreen = ({ navigation }) => {
+  return (
+    <>
+      <Text style={styles.text}>TrackListScreen</Text>
+      <Button title="Go to track detail" onPress={() => navigation.navigate("TrackDetail")} />
+    </>
+  );
+};
+
+const styles = StyleSheet.create({
+  text: {
+    fontSize: 30
+  }
+});
```

Comment:
"tema de casa" :)) - ai repetat stilul asta in fiecare fisier - iti propun sa te gandesti si sa gasesti o metoda prin care sa refolosesti stilul la mod global

Reply:
created reusable styles.js for duplicate styles